discovery-provider/src/tasks/index_solana_plays.py

```diff
@@ -378,7 +378,7 @@ def index_solana_plays(self):
     # Define lock acquired boolean
     have_lock = False
     # Define redis lock object
-    update_lock = redis.lock("solana_plays_lock", blocking_timeout=25)
+    update_lock = redis.lock("solana_plays_lock", blocking_timeout=25, timeout=7200)
```

Comment:
2 hours right? from your experience that'd be enough to cover like any processing here?

Reply:
i think so - especially given how this traverses the program history, but i kind of want to increase it to double just to be safe

Reply:
👍